src/hooks/useWindowSize.ts

```diff
@@ -29,3 +29,8 @@ export function useWindowSize() {
 
   return windowSize
 }
+
+export function useIsDesktopView() {
```

Comment:
navbar has a 1260 breakpoint.  I couldn't find this in the code, so I made this hook.  If there's something better, lmk

Reply:
we really should be trying to do this in css alone ideally. in this case why do you need it as a prop?

Reply:
if the breakpoint changes then it will be annoying to back and change this everywhere.  The other option is we set it as a constant and export it, then use a media query like you are suggesting

Reply:
we should use the constant, there are some in Explore. i'm happy to move them into theme.

Reply:
https://github.com/Uniswap/interface/blob/main/src/components/Tokens/constants.ts